### DISCUSSION TITLE
Is there a way to run this project on my own server?

### DISCUSSION THREAD

**Discussion by leninriv** [pockethost/pockethost#324] [Q&A]
I would like to be able to run this project on my own server with my own domain.
I would like to have my pocketbase projects hosted on my own server

**Comment by benallfree** [comment]
@leninriv Do you need to run more than one PocketBase instance?

**Reply by leninriv** [reply]
Yes. 3 for now. But in the future I will need more. It is posible ?

**Reply by benallfree** [reply]
Yes, that is the design goal of PocketHost (multitenant PocketBase). Please stay tuned for the upcoming CLI tool, it will probably be exactly what you are looking for:)

**Comment by eBizDnA** [comment]
I am also interested to being able to host the picketbase instances and store the data on my own server. Thanks for all your help 🙏

**Reply by ajomuch92** [reply]
See the project Pocketbase itself, they have a guide to run the project locally and also instructions how to deploy
